### ISSUE TITLE
Wrong namespace listed for VariableNumber

### ISSUE THREAD

**Issue by JoeCohen** [bbatsov/rubocop#4668]
The message for a VariableNumber offense incorrectly lists the namespace as Style, rather than Naming.

--------

## Expected behavior

The message displayed for a VariableNumber violation should say: 
>**Naming**/VariableNumber

## Actual behavior

The message actually displayed says: 
>**Style**/VariableNumber
>
Screenshot:
>
![screen shot 2017-08-15 at 07 26 13](https://user-images.githubusercontent.com/45460/29320947-b89918f2-818d-11e7-8848-86c042c82fd8.JPG)

## Steps to reproduce the problem

I think simply including a violation should replicate the problem. E.g.:
```ruby
variable_1 = 1
```
Perhaps additional switches are necessary.  I encounter the error when running:
>$ rubocop -RD -f html > rubocop.html test/controllers/observer_controller_test.rb

for the app in https://github.com/MushroomObserver/mushroom-observer.  
But note that I am in the process of removing the RuboCop violations in that file, so to reproduce the problem in that app, it may be necessary to deliberately add a violation.

## RuboCop version

$ rubocop -V
0.49.1 (using Parser 2.4.0.0, running on ruby 2.2.3 x86_64-linux)

**Comment by pocke** [comment]
VariableNumber cop has been renamed to `Naming/VariableNumber` in master, but it hasn't been released yet. So, the cops says `Style/VariableNumber` in v0.49.1 .

Our documentation mentions about master branch by default.
https://readthedocs.org/projects/rubocop/
![170816094726](https://user-images.githubusercontent.com/4361134/29342819-0de6cb9e-8268-11e7-895b-e1b402f567b2.png)



So, I think we should change the default to `stable`, and build the documentation each release.
The `latest` version is updated each commit in master, but the `stable` version is updated each latest release. http://docs.readthedocs.io/en/latest/versions.html#


@bbatsov Could you change the project setting or invite me to readthedocs? My account name is pocke.

**Comment by JoeCohen** [comment]
@pocke: Thank you so much for the explanation!
